https://issues.redhat.com/browse/ACM-21323 (#8015)
open-cluster-management/rhacm-docs: release_notes/known_issues_application.adoc

// 2.6:25445

The `allow` and `deny` list feature does not work in Object storage application subscriptions.

[#cluster-permission-fails]
== _ClusterPermission_ resource fails when creating multiple role bindings
// 2.14:21342

> If you create a `ClusterPermission` resource that contains multiple role bindings within the same namespace and do not set the optional `name` field, it fails. This error occurs because the same name is used for the role bindings in the namespace.

Reviewer: Optional: Vale advises us to use "different" or "many" instead of multiple. 

```suggestion
If you create a `ClusterPermission` resource that contains different role bindings within the same namespace and do not set the optional `name` field, it fails. This error occurs because the same name is used for the role bindings in the namespace.
```

> If you create a `ClusterPermission` resource that contains multiple role bindings within the same namespace and do not set the optional `name` field, it fails. This error occurs because the same name is used for the role bindings in the namespace.

Author: ```suggestion
If you create a `ClusterPermission` resource that contains many role bindings within the same namespace and do not set the optional `name` field, it fails. This error occurs because the same name is used for the role bindings in the namespace.
```

> The `ClusterPermission` resource fails when you create the resource that contains multiple role bindings within the same namespace and do not set the optional `name` field. This occurs because the same name is used for the role bindings in the namespace.

Reviewer 2: ```suggestion
The `ClusterPermission` resource fails when you create a resource with multiple role bindings for the same namespace and do _not_ set the optional `name` field. This error occurs because the same name is used for the role bindings in the namespace.
```

Author: @swopebe If we say "a" resource, it might confuse users because they might assume that this is for any resource. The `ClusterPermission` resource fails when a user creates a `ClusterPermission` resource with multiple role bindings

@fxiang1 can you help confirm? Does this error occur if any resource has multiple role bindings? 
```suggestion
The `ClusterPermission` resource fails when you create a `ClusterPermission` resource with multiple role bindings for the same namespace and do _not_ set the optional `name` field. This error occurs because the same name is used for the role bindings in the namespace.
```

Reviewer 2: The best way to write this (first to understand it):

If you create a `ClusterPermission` resource that contains multiple role bindings within the same namespace and do not set the optional `name` field, it fails. This occurs because the same name is used for the role bindings in the namespace.

Reviewer 3: Yes what Brandi suggested sounds good!

> The `ClusterPermission` resource fails when you create the resource that contains multiple role bindings within the same namespace and do not set the optional `name` field. This occurs because the same name is used for the role bindings in the namespace.

Author: ```suggestion
If you create a `ClusterPermission` resource that contains multiple role bindings within the same namespace and do not set the optional `name` field, it fails. This error occurs because the same name is used for the role bindings in the namespace.
```
